src/components/header/Hero.js

```diff
@@ -17,42 +23,42 @@ const Hero = ({ bounceInAnimation, executeScroll }) => {
                         fontWeight: '500'
                     }}
                 >
-                    Kromatika DEX Trade
+                    KROMATIKA — next generation DEX Trading, powered by Uniswap and Chainlink
```

Comment:
@sNnickK this is too much text for a header. Maybe a shorter header and then a subheader that is way smaller?